work/COVIDvu-US.ipynb

```diff
@@ -55,8 +53,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%bash\n",
-    "./refreshdata"
+    "%pushd COVID-19\n",
+    "\n",
+    "%sx git pull\n",
```

Comment:
OKi, this is interesting.  Let me check what's up.  Was this crash in the mini lab at https://covidvu.cime.dev ?

Reply:
No crash - did you ensure `refreshdata.config` has all current data needed to run the job?  `refreshdata` does a lot of things, not only pulls the data.